code/modules/admin/banjob.dm

```diff
@@ -140,3 +140,58 @@ DEBUG
 				jobban_savebanfile()
 			return 1
 	return 0
+
+/mob/verb/displayjobbans()

```

Comment:
Seems like slight overkill to me, plus I'd rather keep nanoUI out of this PR for the sake of simplicity.

Another day.